tests/scripts/all.sh

```diff
@@ -185,6 +185,10 @@ pre_initialize_variables () {
     # Include more verbose output for failing tests run by CMake or make
     export CTEST_OUTPUT_ON_FAILURE=1
 
+    # Globally used CFLAGS
```

Comment:
Possibly worth a comment here that these flags are used for all compilers on all builds. (so use carefully)